resources/views/modular/components/events-column.blade.php

```diff
@@ -37,7 +37,7 @@
             @endforeach
             @if($dates == end($data))
                 <li class="flex -mx-2 ml-17">
-                    <a href="//events.wayne.edu/{{ $dates[0]['calendar_url'] ?? 'main/' }}month/" class="hover:underline">{{ $link_text ?? 'More events' }}</a>
+                    <a href="//events.wayne.edu/{{ $component['cal_name'] ?? 'main/' }}month/" class="hover:underline">{{ $component['link_text'] ?? 'More events' }}</a>
```

Comment:
Let's make this `href="//events.wayne.edu/{{ $component['cal_name'] ?? 'main/' }}upcoming"`

"Upcoming" is more likely to have events than "month" and no trailing slash anymore.